lib/kinematics/include/kinematics/matrix_types.h

```diff
@@ -25,6 +25,8 @@ namespace kinematics {
   typedef Eigen::Matrix<double, 7, 7> Matrix7d;
   typedef Eigen::Matrix<double, 8, 8> Matrix8d;
   typedef Eigen::Matrix<double, 9, 9> Matrix9d;
+
```

Comment:
If we only use it in the Rotation3::FromRotationMatrix(), there's no need to add a typedef here.